tests/output/printers/LoggingTest.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import logging
+import json
+
+from coalib.output.Logging import configure_json_logging
+
+
+class LoggingTest(unittest.TestCase):
+
+    def test_json_logging(self):
+        log_stream = configure_json_logging()
+
+        logging.debug('This is debug log.')
+        logging.info('This is info log.')
+        logging.warning('This is warning log.\n This is continued')
+        logging.error('This is error log.')
+        # This is a list of logs dicts.
+        # Each log dict has a `message`, `level` and `time`
+        # We will check message and level for each of the logs
+
```

Comment:
though it's just a comment, you should move the newline above it, as it addresses the `logs_list` variable ;)